english/bite-the-marshmallow/dentist/dentist-office.md

```diff
@@ -1,7 +1,7 @@
 As you walk into the dental office, the pain in your mouth subsides.
 Just as you start feeling happy that your mouth doesn't hurt anymore,
 you look around and realize you're back in that weird room with the marshmallow walls.
-The garishly dressed, vacant-eyed young man is still lounging on the sofa.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.